3rd-party/woocommerce-services.php

```diff
@@ -73,6 +73,9 @@ public function error_notice() {
 	 * @return bool result of installation/activation
 	 */
 	private function install() {
+		$jetpack = Jetpack::init();
+		$jetpack->stat( 'jitm', 'wooservices-activated-' . JETPACK__VERSION );
```

Comment:
Would it make sense to move this below after success, so we're not tracking failed attempts?